CMakeLists.txt

```diff
@@ -76,8 +76,14 @@ set(WPEBACKEND_FDO_PUBLIC_HEADERS
     include/wpe-fdo/fdo.h
 )
 
+set(WPEBACKEND_FDO_EXTENSIONS_PUBLIC_HEADERS
+    include/wpe-fdo/extensions/video-plane-display-dmabuf.h
+    include/wpe-fdo/extensions/video-plane-display-dmabuf-receiver.h
```

Comment:
We are forcing single header inclusion in fdo. I'm not sure if extensions are different or optional to require its own header, or one header per extension.

Reply:
The first header is used for the sender (WebKit) and the second for the consumer (Cog). Technically both headers could be merged I suppose though...

Reply:
In other parts of the lower-level WPE stack (like in `libwpe`) we use a single header in these cases, so I would vote for keeping this consistent with the existing practice.